base/src/main/java/io/spine/annotation/Internal.java

```diff
@@ -33,8 +33,28 @@
 import java.lang.annotation.Target;
 
 /**
- * Annotates a program element (class, method, package, etc.) which is internal to Spine and is
- * not a part of the public API, and thus should not be used by users of the framework.
+ * Annotates a program element (class, method, package, etc.) which is not a part of a public API,
+ * and thus should not be used by people who are not members of the team developing the module
+ * containing this program element.
+ *
+ * <p>If the annotation is used for a constructor, a field, a method, or a package, it means
+ * that corresponding element is internal to the Spine Event Engine and as such should
+ * not be used programmers using the framework.
+ *
+ * <p>If the annotation is used for a type it may mean one of the following:
+ * <ol>
+ *     <li>This type is internal to the Spine Event Engine framework.
+ *
+ *     <li><p>The type is internal to a bounded context, artifact of which exposes the type to
+ *     the outside world (presumably for historical reasons).</p>
+ *
+ *     <p>The type with this annotation can be used only inside the bounded context
```

Comment:
I think some of these tags should be list items, not paragraphs. At least, they are formatted in this way. 

Please review.

Reply:
@armiol, how about now?

Reply:
Looks more consistent to me. Thanks!